packages/renderer/src/lib/pod/PodmanPodDetailsSummary.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+import { Link } from '@podman-desktop/ui-svelte';
+import { router } from 'tinro';
+
+import DetailsCell from '../details/DetailsCell.svelte';
+import DetailsSubtitle from '../details/DetailsSubtitle.svelte';
+import DetailsTable from '../details/DetailsTable.svelte';
+import DetailsTitle from '../details/DetailsTitle.svelte';
+import type { PodInfoUI } from './PodInfoUI';
+
+export let pod: PodInfoUI;
```

Comment:
The typing might be wrong here, as you are checking bellow `{#if pod}` we may prefer to clarify it and express that this component handle the case where pod is undefined.

```suggestion
export let pod: PodInfoUI | undefined;
```